### PR TITLE
sanitize test case of plugin_watcher

### PR DESCRIPTION
**Pull request by tianshapjq**
**What this PR does / why we need it**:
sanitize test case of plugin_watcher

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
none
```

### CODE REVIEW

File: pkg/kubelet/util/pluginwatcher/example_plugin.go [hunk 1]
```diff
@@ -70,11 +70,6 @@ func (s *pluginServiceV1Beta2) RegisterService() {
 	v1beta2.RegisterExampleServer(s.server.grpcServer, s)
 }
 
-// NewExamplePlugin returns an initialized examplePlugin instance
```
Comment (yujuhong) [inline pkg/kubelet/util/pluginwatcher/example_plugin.go]: Why deleting the constructor?

File: pkg/kubelet/util/pluginwatcher/example_plugin.go [hunk 2]
```diff
@@ -108,13 +103,13 @@ func (e *examplePlugin) NotifyRegistrationStatus(ctx context.Context, status *re
 
 // Serve starts a pluginwatcher server and one or more of the plugin services
 func (e *examplePlugin) Serve(services ...string) error {
-	glog.Infof("starting example server at: %s\n", e.endpoint)
+	glog.Infof("starting example server at: %s", e.endpoint)
```
Comment (yujuhong) [inline pkg/kubelet/util/pluginwatcher/example_plugin.go]: s/%s/%q for clarity

File: pkg/kubelet/util/pluginwatcher/example_plugin.go [hunk 3]
```diff
@@ -108,13 +103,13 @@ func (e *examplePlugin) NotifyRegistrationStatus(ctx context.Context, status *re
...
 	lis, err := net.Listen("unix", e.endpoint)
 	if err != nil {
 		return err
 	}
 
-	glog.Infof("example server started at: %s\n", e.endpoint)
+	glog.Infof("example server started at: %s", e.endpoint)
```
Comment (yujuhong) [inline pkg/kubelet/util/pluginwatcher/example_plugin.go]: s/%s/%q for clarity

File: pkg/kubelet/util/pluginwatcher/example_plugin.go [hunk 4]
```diff
@@ -149,7 +144,7 @@ func (e *examplePlugin) Serve(services ...string) error {
 }
 
 func (e *examplePlugin) Stop() error {
-	glog.Infof("Stopping example server at: %s\n", e.endpoint)
+	glog.Infof("Stopping example server at: %s", e.endpoint)
```
Comment (yujuhong) [inline pkg/kubelet/util/pluginwatcher/example_plugin.go]: s/%s/%q for clarity

File: pkg/kubelet/util/pluginwatcher/example_plugin.go (hunk 1 above)
Comment (tianshapjq) [reply pkg/kubelet/util/pluginwatcher/example_plugin.go]: cause no callers for this constructor, only the NewTestExamplePlugin is used.

**Pull request closed without merging**